app/presenters/measure_presenter.rb

```diff
@@ -8,7 +8,7 @@ def has_measure_conditions?
   end
 
   def has_additional_code?
-    additional_code.present?
```

Comment:
Previously:

Additional code model present with code: True
Additional code model present without code: False
Additional code model not present: False

Now:

Additional code model present with code: True
Additional code model present without code: False
Additional code model not present: Exception

Will there always be an AdditionalCode model present?

Reply:
This is true. AFAIK we never iterate over nothing/nil record values so this should be fine

Reply:
I think it was checking the model because a delegator existed for present? from the model to its code attribute value

Reply:
And this was just a byproduct of that